From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@ClarityQueen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-diree). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Commit:
fixing typo

sanjeev3 committed:
Add files via upload

sanjeev3 committed:
Update overview.md

sanjeev3 committed:
Add files via upload

sanjeev3 committed:
Update overview.md

sanjeev3 committed:
Delete credentials.PNG

sanjeev3 committed:
Update try-image-api.md

sanjeev3 committed:
Update try-image-list-api.md

sanjeev3 committed:
Update try-review-api-job.md

sanjeev3 committed:
Update try-review-api-review.md

sanjeev3 committed:
Update try-image-api.md

sanjeev3 committed:
Update try-image-list-api.md

sanjeev3 committed:
Update try-review-api-workflow.md

sanjeev3 committed:
Update try-terms-list-api.md

sanjeev3 committed:
Update try-text-api.md

sanjeev3 committed:
Update credentials.md

sanjeev3 committed:
Add files via upload

sanjeev3 committed:
Update credentials.md

chetansriv committed:
box tutorial update

danieldobalian committed:
Refactored table and some text

danieldobalian committed:
Added some links about v2.0

danieldobalian committed:
Added v1 link in v2 doc

danieldobalian committed:
minor fix

danieldobalian committed:
minor fix

danieldobalian committed:
minor fix

Commit:
edit pass: storage-backup-and-disaster-recovery-for-azure-iaas-disks

GitHubber17 committed:
edit pass: vpn-gateway-3rdparty-device-config-cisco-asa

GitHubber17 committed:
edit pass: vpn-gateway-3rdparty-device-config-overview

anoobbacker committed:
Updating the new scripts

Using ARM SDK

anoobbacker committed:
Updating the new backup script

New SDK based script.

anoobbacker committed:
Adding new script

ARM SDK based scripts

WilliamAntonRohm committed:
edited hdinsight-spark-shell.md and added its media files

WilliamAntonRohm committed:
edits per Acrolinx

WilliamAntonRohm committed:
edits per Acrolinx

WilliamAntonRohm committed:
edits per Acrolinx

WilliamAntonRohm committed:
added hdinsighth-spark-streaming-overview.md and its media files

WilliamAntonRohm committed:
testing updates from original to Dev Guide versions

WilliamAntonRohm committed:
testing updates

WilliamAntonRohm committed:
editing

WilliamAntonRohm committed:
merging in Solliance changes

WilliamAntonRohm committed:
merging in Solliance changes

WilliamAntonRohm committed:
Merge pull request #1 from MicrosoftDocs/master

bring in master

WilliamAntonRohm committed:
Merge pull request #2 from MicrosoftDocs/master

bringin' 'em home!

WilliamAntonRohm committed:
merging in Solliance changes

v-nagta committed:
USER STORY 1075311 SaaS App Tutorial: DATABASICS Update

v-nagta committed:
Merge branch 'master' into databasicsupdate

v-nagta committed:
updated

v-nagta committed:
update

iainfoulds committed:
Adding autoscale commands

iainfoulds committed:
Acrolinx updates

iainfoulds committed:
Revisions

WilliamAntonRohm committed:
heavy editing

tfitzmac committed:
updated portal deploy

tfitzmac committed:
wording change

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into kafka

Commit:
writing

TerryLanfear committed:
new files

mgblythe committed:
Move content into include

YuriDio committed:
new asc articles

WilliamAntonRohm committed:
Merge branch 'master' of https://github.com/WilliamAntonRohm/azure-docs-pr into W-devguide-to-publish

WilliamAntonRohm committed:
Merge pull request #3 from MicrosoftDocs/master

bring in latest ma

WilliamAntonRohm committed:
final edits

tfitzmac committed:
changed image

WilliamAntonRohm committed:
fixed image file name

tfitzmac committed:
added new links in toc and landing

Commit:
updates for live service

Commit:
date

tfitzmac committed:
added links to new topics

WilliamAntonRohm committed:
fixing links to other articles

karthikananth committed:
1506901

karthikananth committed:
1506901 - updated ms.date

mgblythe committed:
Move images, update links

curtand committed:
removed inadvertent user name

curtand committed:
updated date

TerryLanfear committed:
update to existing file

HeidiSteen committed:
JL edits to analyzer doc

daveirwin1 committed:
Added download link

tfitzmac committed:
fixed wording

WilliamAntonRohm committed:
minor fix / test commit

YuriDio committed:
fixing articles

TerryLanfear committed:
updates to existing article

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into sept-datacoll

WilliamAntonRohm committed:
fixed remaining link error

tfitzmac committed:
removed locale from link

tfitzmac committed:
fixing links

mmacy committed:
[aci] minor quickstart update

ShawnJackson committed:
edit pass: hana-overview-high-availability-disaster-recovery

GitHubber17 committed:
Merge pull request #21989 from tfitzmac/0908portaldeploy

updated portal deploy

PRMerger14 committed:
Merge pull request #22218 from tfitzmac/0911eg2

added links to new topics

PRMerger12 committed:
Merge pull request #22234 from daveirwin1/master

Added download link

GitHubber17 committed:
Merge pull request #22221 from karthikananth/routefilterupdate

Route filter documentation update

tfitzmac committed:
removed tutorial link

PRMerger14 committed:
Merge pull request #22248 from mmacy/aci-minor-updates

[aci] minor quickstart update

GitHubber17 committed:
Merge pull request #21941 from iainfoulds/vmlinuxvmsstutorial

Linux VMSS autoscale

WilliamAntonRohm committed:
added two new Dev Guide articles

TerryLanfear committed:
new file

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into septrel_01

CawaMS committed:
explaining the Profiler sampling algorithm and overhead

ShawnJackson committed:
edit pass: hana-overview-high-availability-disaster-recovery

PRMerger20 committed:
Merge pull request #22255 from CawaMS/master

explaining the Profiler sampling algorithm and overhead

Commit:
task 315: update table, edits, add disable HTTPS image

WilliamAntonRohm committed:
revised per PR review comments

Commit:
Merge pull request #3 from MicrosoftDocs/master

Getting latest changes from upstream

curtand committed:
fixed metadata

PRMerger18 committed:
Merge pull request #22231 from HeidiSteen/heidist-analyzer

JL edits to analyzer doc

Commit:
task 315: edits

TerryLanfear committed:
updates per acrolinx

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into septrel_01

jennyhunter-msft committed:
Added Python2 next step link

- Added next step link for Python2 runbooks

jennyhunter-msft committed:
Fix Capitalization and Example for Python

- Removed certificate example from the credential example
- Fixed capitalization from "python" to "Python"
- Added next step for Python

TerryLanfear committed:
fixes

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into sept-pricing

jennyhunter-msft committed:
Additional Python2 edits for variable docs

- Fixed the capitalization of "python" to "Python"
- Fixed incorrect function call
- Added description for functions

jennyhunter-msft committed:
Fix Capitalization of Python in Connection docs

- Fixed capitalization of "python" to "Python"

jennyhunter-msft committed:
Fixed "conection" typo

- Corrected "conection" to "connection"

TerryLanfear committed:
fix

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into sept-onboard

jennyhunter-msft committed:
Removed double "the"

- Removed the second occurrence of "the"

jennyhunter-msft committed:
Update AA certificate docs with Python info

- Added Python2 functions and descriptions
- Added an example of using Python2 to access certificates

jennyhunter-msft committed:
Added hyphen for "third-party"

- Added hyphen for "third-party"

PRMerger18 committed:
Merge pull request #22259 from curtand/name0911

fixed metadata

PRMerger20 committed:
Merge pull request #22224 from curtand/mfa0911

removed inadvertent user name

HeidiSteen committed:
more JL Edits

cychua committed:
Add consistencyguid bug

PRMerger13 committed:
Merge pull request #22269 from cychua/master

Add consistencyguid bug

GitHubber17 committed:
Merge pull request #22213 from tfitzmac/0911eg

added new links in toc and landing

GitHubber17 committed:
Merge pull request #22198 from mgblythe/mb_azure-functions

Move content into include

Commit:
Fix a couple of typos

GitHubber17 committed:
Merge pull request #22208 from WilliamAntonRohm/W-devguide-to-publish

merging in HDInsight Dev Guide files -- first batch

GitHubber17 committed:
Merge pull request #22258 from dksimpson/dean-315

task 315: update table, edits, add disable HTTPS image

PRMerger13 committed:
Merge pull request #21340 from anoobbacker/patch-13

Updating the new backup script

PRMerger13 committed:
Merge pull request #21338 from anoobbacker/patch-14

Adding new script

PRMerger18 committed:
Merge pull request #21339 from anoobbacker/patch-12

Updating the new scripts

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-analyzer

HeidiSteen committed:
rebuild

neilpeterson committed:
add cd

Commit:
Merge pull request #22268 from HeidiSteen/heidist-analyzer

more JL Edits

PRMerger14 committed:
Merge pull request #22273 from neilpeterson/prepare-app-patch

add cd

Commit:
Update ASA limits

PRMerger18 committed:
Merge pull request #22271 from jerrykindall/release-text-analytics-typos

Fix a couple of typos

genlin committed:
add steps for people who cannot see the add-admin option

genlin committed:
change wordin

HeidiSteen committed:
author review

HeidiSteen committed:
changed subscription to access key

HeidiSteen committed:
more subscription key changes

PRMerger15 committed:
Merge pull request #22355 from HeidiSteen/heidist-ta-api

Text analytics -- author change, access key string replacement

PRMerger12 committed:
Merge pull request #22353 from genlin/ci111

billing-add-change-azure-subscription-administrator.md

ggailey777 committed:
Add resources link to Functions videos

Commit:
acrolinx and toc

TerryLanfear committed:
fixes

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into sept-onboard

Commit:
fixing typo

TerryLanfear committed:
update to image

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into septrel_01

PRMerger20 committed:
Merge pull request #22367 from Blackmist/fresh5

fixing typo

YuriDio committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into AugustUpdates_9

YuriDio committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into AugustUpdates_9

HeidiSteen committed:
Ocp-Apim-Subscription-Key revert

YuriDio committed:
fixing article

PRMerger14 committed:
Merge pull request #22369 from HeidiSteen/heidist-ta-api

Ocp-Apim-Subscription-Key revert

TerryLanfear committed:
added TOC changes

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into septrel_01

rjagiewich committed:
Merge pull request #22358 from ggailey777/patch-9

Add resources link to Functions videos

Lisaco88 committed:
Merge pull request #16308 from Blackmist/kafka

writing new kafka document

rjagiewich committed:
Merge pull request #22352 from JSeb225/patch-8

Update ASA limits

Commit:
updated links and add H2s

Commit:
Merge pull request #22371 from stevestein/SqlDbApiLinks

updated links and add H2s

rjagiewich committed:
Merge pull request #22253 from TerryLanfear/sept-onboard

new file

WilliamAntonRohm committed:
Merge pull request #4 from MicrosoftDocs/master

getting in latest

Commit:
edit titles

rjagiewich committed:
Merge pull request #22240 from TerryLanfear/sept-datacoll

Sept datacoll

rjagiewich committed:
Merge pull request #22228 from TerryLanfear/sept-pricing

update to existing file

jennyhunter-msft committed:
Corrected typo "graphical" -> "PowerShell"

- Corrected a typo in the next steps where the link for "My first PowerShell runbook" was labeled as "My first graphical runbook"

Lisaco88 committed:
Merge pull request #22201 from YuriDio/AugustUpdates_9

new asc articles

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into septrel_01

TerryLanfear committed:
fixes to images

PRMerger10 committed:
Merge pull request #22262 from jennyhunter-msft/patch-3

Added Python2 next step link to standalone doc

PRMerger17 committed:
Merge pull request #22263 from jennyhunter-msft/patch-4

Fix Capitalization and Example for Python

Commit:
Merge pull request #22264 from jennyhunter-msft/patch-5

Additional Python2 edits for variable docs

Commit:
Merge pull request #22265 from jennyhunter-msft/patch-6

Fix Capitalization of Python in Connection docs

v-thepet committed:
Merge pull request #22267 from jennyhunter-msft/patch-7

Update AA certificate docs with Python info

BYHAM committed:
sql-database-aad-authentication, updated link.

PRMerger20 committed:
Merge pull request #22379 from jennyhunter-msft/patch-8

Corrected typo "graphical" -> "PowerShell"

singhkays committed:
Update migration-classic-resource-manager-cli.md

sethmanheim committed:
Link to ADLS article on setting permissions

v-alje committed:
Merge pull request #22381 from MicrosoftDocs/FromPublicRepo

From public repo

PRMerger15 committed:
Merge pull request #22386 from singhkays/patch-1

Update migration-classic-resource-manager-cli.md

jennyhunter-msft committed:
Fixed Python code examples

- Removed unnecessary comment in print example
- Fixed indentation for authentication example (Python is very sensitive to indentation and white space)

TerryLanfear committed:
updated articles

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into sept-rec

Commit:
Merge pull request #22390 from jennyhunter-msft/patch-3

Fixed Python code examples

itechedit committed:
Merge pull request #22387 from sethmanheim/cap-up

Link to ADLS article on setting permissions

TerryLanfear committed:
added border to images

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into septrel_01

v-thepet committed:
Merge pull request #19087 from chetansriv/boxupdate

box tutorial update

Commit:
Added a bullet point about performance/scale problems.

sanjeev3 committed:
Add files via upload

sanjeev3 committed:
Update connectors.md

MicrosoftGuyJFlo committed:
Update to SaaS Landing Page

Commit:
edit titles

sanjeev3 committed:
Update toc.yml

itechedit committed:
Merge pull request #22123 from TerryLanfear/septrel_01

new files

MicrosoftGuyJFlo committed:
Update active-directory-saas-box-tutorial.md

itechedit committed:
Merge pull request #22401 from MicrosoftGuyJFlo/SaaSTutorialUpdates

Update to SaaS Landing Page

douglaslMS committed:
Added missing parenthesis.

sanjeev3 committed:
Add files via upload

sanjeev3 committed:
Update overview.md

PRMerger12 committed:
Merge pull request #22402 from MicrosoftGuyJFlo/patch-2

Update active-directory-saas-box-tutorial.md

sanjeev3 committed:
Delete credentials3.PNG

sanjeev3 committed:
Add files via upload

sanjeev3 committed:
Delete credentials3.PNG

sanjeev3 committed:
Add files via upload

Commit:
Merge pull request #22404 from douglaslMS/sync-fix-syntax

Added missing parenthesis.

sanjeev3 committed:
Delete credentials3.PNG

sanjeev3 committed:
Add files via upload

TerryLanfear committed:
another article

sanjeev3 committed:
Update api-reference.md

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into sept-rec

sanjeev3 committed:
Update api-reference.md

sanjeev3 committed:
Add files via upload

sanjeev3 committed:
Update credentials.md

sanjeev3 committed:
Update Text-Reviews.md

v-thepet committed:
remove nonexistent bookmark

TerryLanfear committed:
update to article

PRMerger17 committed:
Merge pull request #22408 from v-thepet/newlinks

production bug: remove nonexistent bookmark

Commit:
acrolinx

ShawnJackson committed:
Update storage-backup-and-disaster-recovery-for-azure-iaas-disks.md

Lisaco88 committed:
Merge pull request #22391 from TerryLanfear/sept-rec

Sept rec

ShawnJackson committed:
Merge pull request #20913 from LoriMerrick/storage-backup-and-disaster-recovery-for-azure-iaas-disks-2

edit pass: storage-backup-and-disaster-recovery-for-azure-iaas-disks

Commit:
Improved wording

ShawnJackson committed:
Update vpn-gateway-3rdparty-device-config-cisco-asa.md

Ja-Dunn committed:
Merge pull request #20394 from danieldobalian/master

Refactoring the library guides

pharring committed:
Update author

PRMerger15 committed:
Merge pull request #22375 from stevestein/SqlDbSaasTitles

edit titles

Ja-Dunn committed:
Merge pull request #22251 from ShawnJackson/hana-overview-high-availability-disaster-recovery

edit pass: hana-overview-high-availability-disaster-recovery

PRMerger13 committed:
Merge pull request #21905 from v-nagta/databasicsupdate

USER STORY 1075311 SaaS App Tutorial: DATABASICS Update

PRMerger16 committed:
Merge pull request #22399 from piotrci/GBLChanges

Updated GBL limitations section

PRMerger14 committed:
Merge pull request #22385 from BYHAM/patch-15

sql-database-aad-authentication, updated link.

pharring committed:
Remove instructions for pre-release ASP.NET Core 2.0 applications

Ja-Dunn committed:
Merge pull request #22407 from TerryLanfear/sept-os

update to article

WilliamAntonRohm committed:
merging Dev Guide updated content into master

Lisaco88 committed:
Merge pull request #22406 from sanjeev3/master

Updates to the Credentials screen.

PRMerger17 committed:
Merge pull request #22417 from pharring/master

Remove pre-release instructions

ShawnJackson committed:
Update vpn-gateway-3rdparty-device-config-overview.md

ShawnJackson committed:
Merge pull request #21331 from GitHubber17/vpn-gateway-articles

edit pass: vpn-gateway articles

CawaMS committed:
Adding Profiler to the Diagnose section

CawaMS committed:
update screenshot on Trace Viewer

Ja-Dunn committed:
Merge pull request #22418 from WilliamAntonRohm/W-sept12-devguide-to-publish

updating pre-existing HDInsight files with new Dev Guide additions

Ja-Dunn committed:
Merge pull request #22422 from CawaMS/master

Adding Profiler to the Diagnose section
